lib/rules/jsx-sort-props.js

```diff
@@ -382,6 +418,9 @@ module.exports = {
         reservedFirst: {
           type: ['array', 'boolean'],
         },
+        customPropsFirst: {
+          type: ['array', 'boolean'],
+        },
```

Comment:
`true` probably shouldn't be an allowed value here - also, the array should be limited to a unique list of strings.
```suggestion
        customPropsFirst: {
          oneOf: [
            {
              type: 'array',
              uniqueItems: true,
              items: { type: 'string' },
            },
            {
              type: 'boolean',
              enum: [false],
            },
          ],
        },
```